lib/utils.pm

```diff
@@ -579,24 +579,21 @@ sub zypper_ar {
     my $params       = $args{params}       // '';
     my $no_gpg_check = $args{no_gpg_check} // '';
 
-    $no_gpg_check = $no_gpg_check ? "--no-gpgcheck" : "";
-    my $prioarg = defined($priority) && !is_sle('<12') ? "-p $priority" : "";
-    my $cmd_ar  = "--gpg-auto-import-keys ar -f $prioarg $no_gpg_check $params $url";
-    my $cmd_mr  = "mr -p $priority $url";
+    $priority     = defined($priority) ? "-p $priority"  : "";
+    $no_gpg_check = $no_gpg_check      ? "--no-gpgcheck" : "";
+    my $cmd_ar  = "--gpg-auto-import-keys ar -f $priority $no_gpg_check $params $url";
```

Comment:
could you kindly explain why is needed this change in this file for this scenario, I cannot find the reason to touch this library and do it without adding and additional commit for it?

Reply:
this is not me. i remember doing a rebase after removing my change there. let me check

Reply:
i reverted it. Sorry about that